README.md

```diff
@@ -732,6 +777,41 @@ mappings to allow automatic CRS detection by `OpenLayers` here.
 -   `useDefaultMappings` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** Whether the default CRS should be mapped
       as well or not. Default is true. (optional, default `true`)
 
+#### toDms
+
+Converts geographic coordinates given in DDD format like `DD.DDDD°` to
+the degree, minutes, decimal seconds (DMS) format like
+`DDD° MM' SS.SSS"`.
+
+##### Parameters
+
+-   `value` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** Value to be converted.
+
+Returns **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Converted value.
+
+#### toDmm
+
+Converts geographic coordinates given in DDD format like `DD.DDDD°` to
+the degree, decimal minutes (DMM) format like `DDD° MM.MMMM`.
+
+##### Parameters
+
+-   `value` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** Value to be converted.
+
+Returns **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Converted value.
+
+#### zerofill
+
+Adds leading zero to all values less than 10 and returns this new
+zerofilled value as String. Values which are greater than 10 are not
+affected.
+
+##### Parameters
+
+-   `value` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** Value to be zerofilled.
+
+Returns **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** converted value with leading zero if necessary.
+
```

Comment:
Docs are available [here](https://terrestris.github.io/ol-util/latest/index.html) and will be updated automatically. I suggest to delete the docs from the README completely (in a follow up MR).

Reply:
Ok, will be done in a separate MR

Reply:
See #151